charts/minibroker/values.yaml

```diff
@@ -1,6 +1,6 @@
 # Default values for the minibroker
 # Image to use
-image: osbkit/minibroker:latest
+image: cap/minibroker:latest
```

Comment:
This should be built from `.kube.organization`, not hard coded in `image` (it needs to add `kube.registry.*` in front too).

Also, we should probably default this to docker hub, and then change it to `registry.suse.com` in the build process (so that non-us people can have a reasonable experience out of the box when they push).